x-pack/docs/en/security/auditing/output-logfile.asciidoc

```diff
@@ -3,13 +3,39 @@
 === Logfile audit output
 
 The `logfile` audit output is the default output for auditing. It writes data to
-the `<clustername>_access.log` file in the logs directory.
+the `<clustername>_audit.log` file in the logs directory. To maintain
+compatibility with releases prior to 6.5.0, another file `<clustername>_access.log`
+is also generated next to it. They differ in the output format but the contents
+are similar. For systems that are not ingesting the audit file for search or
+analytics it is strongly recommended to only keep the newer format.
+Turning off the deprecated output format can be achieved by disabling the logger
+in the `log4j2.properties` file in `ES_PATH_CONF` (hint: there is a config comment
+about it).
 
 [float]
 [[audit-log-entry-format]]
 === Log entry format
 
-The format of a log entry is:
+This is describing the format for log entries as they are appended to the
```

Comment:
I'd change this introductory sentence as follows:

"The log entries in the `<clustername>_audit.log` file have the following format:"